content/02.body.md

```diff
@@ -1,13 +1,23 @@
 ## Introduction
 
-Networks contain information about relationships between entities ("edges between nodes").
-A node's degree is the number of relationships it has in the network.
+Networks contain information about relationships between entities (referred to here as "edges" between "nodes").
+A node's degree is the number of edges it has in the network.
+`TODO: do we need to discuss in-degree vs out-degree and hetnet degree here?`
 Networks contain many nodes, whose degrees can be aggregated to form the network's degree distribution.
-Because different nodes can have very different degrees, real networks have highly variable degree distributions (Figure {@fig:hetionet}).
+Because different nodes can have very different degrees, real networks generally have highly variable degree distributions [@doi:10.1126/science.286.5439.509; @doi:10.1038/s41467-019-08746-5; @doi:10.1371/journal.pone.0017645].
```

Comment:
Below there is the following sentence:

```
Real networks have a variety of degree distributions (Figure {@fig:hetionet}), and they commonly exhibit degree imbalance [@doi:10.1371/journal.pone.0017645; @doi:10.1007/978-1-61779-361-5_13; @doi:10.1038/s41467-019-08746-5; @doi:10.1126/science.286.5439.509].
```

I wonder if we should introduce this concept much earlier like here?

Reply:
I don;t understand this comment. The suggestion is the PR. The blockquoted sentence is from below and currently is in manuscript.

Reply:
My bad, didn't remember that the blockquote was still included below.